### DISCUSSION TITLE
how to designate the right camera

### DISCUSSION THREAD

**Discussion by Sceptersax** [mhaowork/amblegpt#15] [General]
It looks like i got the application running and i do see in the logs a description from Chatgpt about the camera view. This is great! The only problem is, it's the wrong camera. I designated "Hall_cam" in amblegpt config as the camera, just like in frigate config, but it was taking the feed from a different camera. How do i insure that the correct cam stream will be used? Is there something in HA i need to change?

And btw, in HA, I see that "Hall_cam" is "Hall cam". Is the underscore a problem? or maybe that's unrelated.

**Comment by mhaowork** [comment]
Were you able to figure it out? Sorry for the late comment, just noticed this thread. I believe the camera name should match. For example, these are what I had in the Frigate notification configuration in Home Assistant: 
![image](https://github.com/mhaowork/amblegpt/assets/8702853/cd6b8cc1-bf4a-4b56-bdf2-ad1697e05e35)

**Reply by Sceptersax** [reply]
I ended up just disabling all cameras except one just to test and I gotta say, the API works great but chatgpt is so politically correct, I can't use it for my usecase. It won't comment on race, probably not gender.. basically any descriptions about a person besides maybe their actions, are off the table. 

What do you think about Grok?
